pkgs/tools/networking/bitmask-vpn/default.nix

```diff
@@ -78,6 +78,9 @@ buildGoModule rec {
     substituteInPlace providers/vendor.conf \
       --replace "provider = riseup" "provider = ${provider}"
 
+    substituteInPlace $out/share/applications/${provider}-vpn.desktop \
```

Comment:
`$out` is not created yet when in `postPatch`, so this does not work, instead try to substitute in `branding/templates/debian/app.desktop-template`

Reply:
Ah yes my bad, I moved it from `postInstall` and forgot to fix the path. Sorry for the silly mistake and thanks for catching it!